velodrome/grafana-stack/prometheus.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prometheus
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: prometheus
+    spec:
+      containers:
+      - name: prometheus
+        command:
+        image: prom/prometheus:v1.1.3
+        ports:
+        - name: prometheus-port
+          containerPort: 9090
+        volumeMounts:
+        - mountPath: /prometheus
+          name: database-volume
+        - mountPath: /etc/prometheus
+          name: prometheus-config
+      volumes:
+      - name: database-volume
+        persistentVolumeClaim:
+          claimName: prometheus-database-claim
+      - name: prometheus-config
+        configMap:
+          name: prometheus
+          items:
+          - key: prometheus.yml
+            path: prometheus.yml
```

Comment:
can we maybe name this something else? having a `prometheus.yaml` and `prometheus.yml` in the same directory is pretty confusing.

also, it might be nice to have a basic config that at least scrapes something.